src/main/java/seedu/address/ui/UnflippedReviewCard.java

```diff
@@ -47,9 +46,9 @@ public UnflippedReviewCard(Card card) {
 
         answer.setText(EMPTY_STRING);
 
-        card.getTags().stream()
-                .sorted(Comparator.comparing(tag -> tag.tagName))
-                .forEach(tag -> tags.getChildren().add(new PersonCard.CardTag(tag.tagName)));
+        if (!card.getTag().tagName.equals(Tag.TagName.UNTAGGED)) {
```

Comment:
OOps Im supposed to have deleted this class HAHA sorry I missed this out. I dont think its being used now